pkg/networkservice/common/monitor/monitor_connection_server.go

```diff
@@ -22,24 +22,31 @@ import (
 	"github.com/edwarnicke/serialize"
```

Comment:
In current solution we actually dont need changes in network service/monitor pkg. Please revert pkg/networkservice/monitor changes.